doc/releases/release-notes-2.4.rst

```diff
@@ -46,6 +46,13 @@ API Changes
   ignore this diagnostic result or initiate another feed operation
   later.
 
+* The :c:func:`fs_open` function now accepts open flags that are passed as
+  a third parameter.
+  All custom file system front-ends require change to the implementation
+  of ``open`` callback to accept the new parameter.
+  To maintain original behaviour within user code, two argument invocations
+  should be converted to accept third parameter of the value  ``FS_O_CREATE | FS_O_RDWR``.
```

Comment:
"...should be converted to ~accept~*pass a* third ~parameter of the value~*argument*  ``FS_O_CREATE | FS_O_RDWR``."

Reply:
Done.